packages/odata-common/src/entity-builder.ts

```diff
@@ -44,11 +44,11 @@ export class EntityBuilder<
   EntityT extends EntityBase,
   DeSerializersT extends DeSerializers
 > {
-  protected entity: EntityT;
+  protected _entity: EntityT;
```

Comment:
[req] Please also adjust the test service spec, so we can see how the clients will be generated as well as the compilation.

Reply:
[req] I think we do not need the `_` andymore if it is in the reserved words right?

Reply:
I personally would keep this change so by default, a user would not have `entity_1`.

Reply:
@marikaner , as a PO, you have a task for your opinion.

Reply:
I adjusted the code and also made one more manual test with the spec file provided by the users and worked with the `_entity`